README.md

```diff
@@ -1,11 +1,80 @@
 # Description
+
 Coforma's response to the SWIFT technical challenge
 
-## Local Setup
+## TODO: Table of Contents
+
+- TODO: Links here
```

Comment:
Do we want to fill this out in this pull request?

Reply:
Meh I'd rather have Dan do it in a future one.